File: C/PRAK302-2210817310009-MarioFrancaWijaya-.c
#include <stdio.h>

int main(){
    int bil;
    
    printf("Masukan nilai kamu :");
    scanf("%d", &bil);

    if(bil >= 80){
        printf("Nilai kamu setelah di konversi adalah A");
    } else if (bil >= 70 && bil <= 79){
        printf("Nilai kamu setelah di koversi adalah B");
    } else if (bil >= 60 && bil <= 69){
        printf("Nilai kamu setelah di koversi adalah C");
    } else if (bil >= 50 && bil <= 59){
        printf("Nilai kamu setelah di koversi adalah D");
    } else {
        printf("Nilai kamu setelah di konversi adalah E");
    }

    return 0;
}

File: Python/PRAK302-2210817310009-Mario Franca Wijaya.py
bil = int(input("Masukan nilai kamu:"))

if bil >= 80:
    print("Nilai kamu setelah di konversi adalah A")
elif bil >= 70 and bil <= 79:
    print("Nilai kamu setelah di konversi adalah B")
elif bil >= 60 and bil <= 69:
    print("Nilai kamu setelah di konversi adalah C")
elif bil >= 50 and bil <= 59:
    print("Nilai kamu setelah di konversi adalah D")
else :
    print("Nilai kamu setelah di konversi adalah E")